Sources/SwiftGRPC/Core/Channel.swift

```diff
@@ -75,17 +75,18 @@ public class Channel {
   /// Initializes a gRPC channel
   ///
   /// - Parameter address: the address of the server to be called
-  /// - Parameter certificates: a PEM representation of certificates to use
+  /// - Parameter certificates: a PEM representation of certificates to use. If nil, use defaults.
   /// - Parameter clientCertificates: a PEM representation of the client certificates to use
   /// - Parameter clientKey: a PEM representation of the client key to use
   /// - Parameter arguments: list of channel configuration options
-  public init(address: String, certificates: String, clientCertificates: String? = nil, clientKey: String? = nil, arguments: [Argument] = []) {
+  public init(address: String, certificates: String?, clientCertificates: String? = nil, clientKey: String? = nil, arguments: [Argument] = []) {
```

Comment:
`certificates: String? = nil`, or (if that compiles, please try it and let me know if it doesn't) `certificates: String = roots_pem()`.

Reply:
@MrMage 
`roots_pem()` returns an optional, so we have to either force unwrap `roots_pem()` either leave argument optional: `certificates: String? = roots_pem()`
I'm not sure which way better. Looking forward to your decision.

Reply:
Thanks for the explanation! I think we should change `roots_pem()` to forcibly unwrap the results of its two calls instead, at which point it wouldn't need to return an optional. Would you mind adding that?

Reply:
@MrMage 

> Would you mind adding that?
Sure, thank you.

> `certificates: String? = nil`, or (if that compiles, please try it and let me know if it doesn't)
It doesn't.
We should change `roots_pem()` access modifier to `public` to use it as a default value.
To be more specific, compiler fails with following error:
`Global function 'roots_pem()' is internal and cannot be referenced from a default argument value`.

If this way is inappropriate we can also do something like that to hide roots_pem:
```
enum RootCertificates {
  case custom(String)
  case `default`
}
```
But making `roots_pem()` public looks better to me.

Reply:
I think that we should definitely make `roots_pem()` public.

Reply:
Ok, so `Roots.swift` is a generated file which should not be edited according to comment in it.
I suggest to add a static method to `gRPC` class like that:

```
public final class gRPC {
    
  /// Default root certificates provided by gRPC
  public static let defaultRootCertificates = roots_pem()!

...
```

And after that, all should be ok finally.

Reply:
That file can be edited if you edit https://github.com/grpc/grpc-swift/blob/master/Sources/RootsEncoder/main.swift at the same time ;-)

Reply:
Got it! New commit ready for review.